config/routes.rb

```diff
@@ -229,5 +229,5 @@
   get "/:platform/:name.about", to: "projects#about", as: :about_project, constraints: { name: /.*/ }
   get "/:platform/:name.ABOUT", to: "projects#about", constraints: { name: /.*/ }
   get "/:platform/:name", to: "projects#show", as: :project, constraints: { name: /.*/ }, defaults: { format: "html" }
-  get "/:id", to: "platforms#show", as: :platform
+  get "/:id", to: "platforms#show", as: :platform, constraints: { format: "html" }
```

Comment:
Not strictly upgrade related, but something that emerged from the last upgrade. Because this is a root level catch-all route, we end up running a lot of strange/spam/probe requests through this action. 
We don't render any other formats on this action, and if it's supposed to support platform names with a dot in them, it doesn't currently anyway. So this should just be turning CSRF/UnknownFormat errors into 404s as they should be.